.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+title: ''
```

Comment:
Does this actually do anything? If it's empty, it's the default anyway I assume, so we shouldn't set it here?

Reply:
@felixarntz I actually didn't touch the other templates, so it looks like the commit just pulled them all in by default instead of just the new one. I don't know if this can be removed; let me check.